README.md

```diff
@@ -0,0 +1,10 @@
+# Winter school lectures for URSSI 2024
+
+
+## Author
+Eva Maxfield Brown
+
+
```

Comment:
You should also include information on the software license and a description of the software.